packages/mint-components/src/components/sqm-instant-access-registration/sqm-instant-access-registration.feature

```diff
@@ -0,0 +1,104 @@
+@author:truman
+Feature: Instant access referrer registration
+
+  @motivating
+  Scenario: User's must provide an email to register
+    Given a user is trying to register as a referrer
+    When they do not provide an email
+    And they submit the form
+    Then an error appears telling the user to provide their email
+    When they provide an email
+    And they submit the form
+    Then they are logged in to the instant access widget
+
+  @motivating
+  Scenario: Users are notified if registration fails
+    Given a user is trying to register
+    And they have included their email
+    When an error occurs with the registration
+    Then an alert banner appears letting the user know about the error
+
+  @motivating
+  Scenario: Error banner is shown if an error occurs during registration
+    Given a user has entered a valid email
+    And they submit the form
+    When an error occurs during the user upsert
+    Then an error banner is displayed
+    And the error banner is at the top of the form
+
+  @motivating
+  Scenario: Successful registration upserts a user
```

Comment:
Is there any state shown while the form is being submitted? Fields and buttons disabled etc?